File: src/class/Enofilo.js
export default class Enofilo {
  constructor(usuario, nombre, apellido, imagenPerfil) {
    this.usuario = usuario;
    this.nombre = nombre;
    this.apellido = apellido;
    this.imagenPerfil = imagenPerfil;
  }

  getNombre() {
    return this.nombre;
  }

  setNombre(nombre) {
    this.nombre = nombre;
  }

  getApellido() {
    return this.apellido;
  }

  setApellido(apellido) {
    this.apellido = apellido;
  }

  getImagenPerfil() {
    return this.imagenPerfil;
  }

  setImagenPerfil(imagenPerfil) {
    this.imagenPerfil = imagenPerfil;
  }
}

File: src/com/BonVino/proyect/Enofilo.java
package com.BonVino.proyect;

public class Enofilo {
    private String usuario;
    private String nombre;
    private String apellido;
    private String imagenPerfil;

    // Constructor
    public Enofilo(String usuario, String nombre, String apellido, String imagenPerfil) {
        this.usuario = usuario;
        this.nombre = nombre;
        this.apellido = apellido;
        this.imagenPerfil = imagenPerfil;
    }

    // Métodos getters y setters
    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getImagenPerfil() {
        return imagenPerfil;
    }

    public void setImagenPerfil(String imagenPerfil) {
        this.imagenPerfil = imagenPerfil;
    }
}
